leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/LeshanServerDemo.java

```diff
@@ -163,8 +161,21 @@ public static void main(String[] args) throws FileNotFoundException {
                 "The path to your server certificate file.\n"
                         + "The certificate Common Name (CN) should generally be equal to the server hostname.\n"
                         + "The certificate should be in X509v3 format (DER encoding).");
+
+        final StringBuilder trustStoreChapter = new StringBuilder();
+        trustStoreChapter.append("\n .");
+        trustStoreChapter.append("\n URI format: file://<path-to-trust-store-file>#<hex-strore-password>#<alias-pattern>");
+        trustStoreChapter.append("\n .");
+        trustStoreChapter.append("\n Where:");
+        trustStoreChapter.append("\n - path-to-trust-store-file is path to pkcs12 trust store file");
```

Comment:
it seems that californium API support more than pkcs12.
Should we be more specific, by listing [all supported format](https://github.com/eclipse/californium/blob/2.4.1/element-connector/src/main/java/org/eclipse/californium/elements/util/SslContextUtil.java#L578) ?

Reply:
Never heard anyone using BKS. So perhaps not worth to mention.

JKS is Java KeyStore and is deprecated in favour of PKCS12 key store in Java tooling. But it is still known by people so we could add that to the list.

Perhaps:
```
path-to-trust-store-file is path to PKCS#12 (.p12) or Java trust store (.jks) file
```
Problem with PEM is that it is just encoding and is often not named as .pem like in Ubuntu CA certificates bundle (PEM) is named:
```
/etc/ssl/certs/ca-certificates.crt
```

Therefore in the config file example I used type to specify what it is. Often there is either ca path setting or ca file setting.

Reply:
Of about PEM certificate bundles -- there is some helpers in certificate chain support code (SecurityUtil.certificateChain) that could be adapted to easier file path variant to load PEM (or DER) files so we could change to that kind of functionality instead of using SslContextUtil.loadTrustedCertificates() which is a bit clumsy.

What one would change for this use case is also support loading of other kind of certificates than EC.

Reply:
Ok let's start with just PKCS#12 in documentation.